roles/oauth/templates/config.json.j2

```diff
@@ -337,6 +337,15 @@
       "redirectUri": "",
       "trusted": true,
       "canGrant": true
+    },
+    {
+      "id": "1a118eae25503e99",
+      "hashedSecret": "19a305e07dc4fdc502321f82745fb0c0de80d31cfa26976ad1117f898cbd1c81",
+      "name": "Mozilla IAM",
+      "imageUri": "",
+      "redirectUri": "http://localhost/",
```

Comment:
what should this be?

Reply:
@gdestuynder do you want to start using e.g. https://auth-dev.mozilla.auth0.com/login for integration with our dev environments?